src/main/java/com/microsoft/sqlserver/jdbc/KerbAuthentication.java

```diff
@@ -94,8 +96,22 @@ private void initAuthInit() throws SQLServerException {
                 Subject currentSubject;
                 KerbCallback callback = new KerbCallback(con);
                 try {
-                    java.security.AccessControlContext context = java.security.AccessController.getContext();
-                    currentSubject = Subject.getSubject(context);
+
+                    try {
+                        java.security.AccessControlContext context = java.security.AccessController.getContext();
+                        currentSubject = Subject.getSubject(context);
+
+                    } catch (UnsupportedOperationException ue) {
+                        if (authLogger.isLoggable(Level.FINE)) {
```

Comment:
Would it be needed ( or good to have) to also check the exact exception message as part of the UnsupportedOperationException ?

Reply:
https://bugs.java.com/bugdatabase/view_bug?bug_id=8328643
>When the Security Manager is removed in a future release, the `Subject.getSubject(AccessControlContext)` method will be degraded further to throw `UnsupportedOperationException` unconditionally

Should be ok, since when an UnsupportedOperationException is thrown, it means security manager deprecation.